daml-lf/transaction/BUILD.bazel

```diff
@@ -48,6 +48,8 @@ da_scala_library(
     srcs = glob(["src/main/**/*.scala"]),
     scala_deps = [
         "@maven//:org_scalaz_scalaz_core",
+        "@maven//:org_typelevel_cats_core",
```

Comment:
do we really need cats for this? I don’t have strong feelings on which one and actually would prefer cats overall but using both scalaz & cats in the same library seems a bit odd.

Reply:
Removed again following Remy's suggestions.